src/libs/actions/User.ts

```diff
@@ -728,6 +728,18 @@ function validateSecondaryLogin(loginList: OnyxEntry<LoginList>, contactMethod:
             value: {isLoading: false},
         },
     ];
+    // Sometimes we will also need to reset the validateCodeSent of ONYXKEYS.VALIDATE_ACTION_CODE in order to receive the magic next time we open the ValidateCodeActionModal.
```

Comment:
Since our style guide usually mention to leave an empty line before comments, just a small suggestion:

```suggestion

    // Sometimes we will also need to reset the validateCodeSent of ONYXKEYS.VALIDATE_ACTION_CODE in order to receive the magic next time we open the ValidateCodeActionModal.
```